tests/unit/backend/corpora/fixtures/database/__init__.py

```diff
@@ -98,11 +100,17 @@ def _create_test_datasets(self):
             id="test_project_dataset_id",
             project_id="test_project_id",
             project_status=ProjectStatus.LIVE.name,
-            dataset_id="test_dataset_id",
+            dataset_id=test_dataset_id,
         )
         self.db.session.add(project_dataset)
         self.db.session.commit()
 
+        deployment_directory = DbDeploymentDirectory(
+            id="test_deployment_directory_id", dataset_id=test_dataset_id, environment="test", url="test_url"
+        )
+        self.db.session.add(deployment_directory)
+        self.db.session.commit()
```

Comment:
Since you've added a `commit` here, you could delete line 106 so that a single transaction occurs within this single function.

Reply:
fixing